src/test/scala/tech/cryptonomic/conseil/tezos/TezosTypesTest.scala

```diff
@@ -29,4 +35,33 @@ class TezosTypesTest extends WordSpec with Matchers {
 
   }
 
+  "should modify parameters with monocle's lenses" in {
+    // given
+    val block = Block(blockData, List(operationGroup.copy(contents = List(origination, transaction.copy(parameters = Some(Micheline("micheline script")))))))
+
+    // when
+    val result = parametersLense.modify(_.map(_ => Micheline("new micheline script")))(block)
```

Comment:
Side note: I guess this `.modify(_.map(...))` pattern could be resolved by defining a more precise optic type, yet I would not dedicate time for it in this PR